app/styles/constants.css

```diff
@@ -30,3 +30,9 @@
     --small-border-radius: 6px; /* smaller elements such as buttons */
     --large-border-radius: 12px; /* larger elements such as panels */
 }
+
+/* ------ MISC PROPERTIES ------ */
+
+:root {
+    --line-height: 1.6;
```

Comment:
Similar issue here. This variable is barely used?

Reply:
Removed it